### DISCUSSION TITLE
No layout edit if need reset password account via module

### DISCUSSION THREAD

**Discussion by Mika17420** [joomla/joomla-cms#39282] [General]
if a user connects via the module and if joomla params for reset password on first login. There is no redirect to the layout=edit page. You have to click on any link to see the page appear

**Comment by Mika17420** [comment]
no one has any solutions to offer?

**Comment by brianteeman** [comment]
Its a bug

**Reply by brianteeman** [reply]
I have opened an issue see #39300
